### ISSUE TITLE
CI job fails because it requires golang 1.13

### ISSUE THREAD

**Issue by colega** [GoogleCloudPlatform/cloudsql-proxy#344]
[Latest master commit] upgraded some deps and the CI job [now fails] because it requires golang 1.13 errors API: 

```
# bazil.org/fuse
../../../bazil.org/fuse/fuse.go:410:5: undefined: errors.As
```

[Latest master commit]: https://github.com/GoogleCloudPlatform/cloudsql-proxy/commit/eb5e09ef70a515db75cc59caaee50d29638ad5db
[now fails]: https://source.cloud.google.com/results/invocations/74a8ad03-4845-4c27-834f-01782d17c5c7/targets/cloud-devrel%2Fcloud-sql%2Fproxy%2Fgo112%2Fperiodic/log

**Comment by kurtisvg** [comment]
I believe this only effects the lint job that's set up - the Dockerfile still works (since it's pinned to '1') and the the dialer doesn't use fuse, folks should be able to continue using it with Go 1.12. 

If this isn't the case please let me know.

**Comment by colega** [comment]
No, you can't use it with Go 1.12 anymore:

```
$ docker run --rm -it golang:1.12 bash
root@78175cb77fca:/go# go get github.com/GoogleCloudPlatform/cloudsql-proxy/cmd/cloud_sql_proxy
# bazil.org/fuse
src/bazil.org/fuse/fuse.go:410:5: undefined: errors.As
```

**Comment by kurtisvg** [comment]
You can't use the proxy, but you can still use the go dialier from Go 1.12 (as it doesn't require fuse):

```
$ docker run --rm -it golang:1.12 bash
root@1790d4992a37:/go# go get github.com/GoogleCloudPlatform/cloudsql-proxy/proxy/proxy
root@1790d4992a37:/go# go get github.com/GoogleCloudPlatform/cloudsql-proxy/proxy/dialers/mysql 
```

**Comment by colega** [comment]
Yes, you still can do that 👍

**Comment by kurtisvg** [comment]
This was fixed in #367
